addon/components/acmidm-login.js

```diff
@@ -1,4 +1,21 @@
-import templateOnly from '@ember/component/template-only';
+import Component from '@glimmer/component';
+import { deprecate } from '@ember/debug';
 
-const AcmidmLoginComponent = templateOnly();
-export default AcmidmLoginComponent;
+export default class AcmidmLoginComponent extends Component {
+  constructor() {
+    super(...arguments);
+
+    deprecate(
+      'The <AcmidmLogin> component is deprecated. Please use the <Acmidm::Login> provider component instead.',
+      false,
+      {
+        id: '@lblod/ember-acmidm-login.acmidm-login.component',
+        for: '@lblod/ember-acmidm-login',
+        since: {
+          enabled: '2.0.0',
```

Comment:
If the previous PR wasn't considered a breaking change, this probably needs to be changed to `1.3.0` instead.